src/plugins/intel_gna/optimizer/gna_pass_manager.cpp

```diff
@@ -1719,6 +1720,29 @@ void UnrollTIPass::run() {
     }
 }
 
+class GnaConstTransformer : public ConstTransformer {
```

Comment:
Could you please add a description for this class, what is the purpose, and how it should be used

Reply:
OK